sdks/java/core/build.gradle

```diff
@@ -117,6 +132,113 @@ task packageTests(type: Jar) {
   classifier = "tests"
 }
 
+class ValidatesRunnerConfig {
+  // Task name prefix
+  String testPrefix
+  // Runner name in documentation string
+  String runnerName
+  // List of test categories to exclude from this task. Optional.
+  List<String> excludes
+  // Pipeline options command line arguments.
+  // TODO: Can option keys be specified multiple times?
```

Comment:
Option keys cannot be specified multiple times.

Reply:
Removed.